components/clamav/files/usr/bin/clamav-enable.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
```

Comment:
Not always that we install the package we actually want the service activated (e.g. usability starts with tens of megabytes of virusdb download - not something I'd want forced onto me when I'm on cellular internet, and maybe from every zone that has a copy installed for whatever reason such as testing-staging, but could share the same db with some sysadmin creativity). Also perhaps I don't want the daemons running at all and a CLI tool suffices for a certain installation.